### PR TITLE
ppm: Update ppm to commit d9bcff111146547e1f4dec63

### PR DESCRIPTION
**Pull request by DeeDeeG**
Bumps `ppm` to a newer copy.

Includes the following ppm pull requests:

- https://github.com/pulsar-edit/ppm/pull/131
- https://github.com/pulsar-edit/ppm/pull/133
- https://github.com/pulsar-edit/ppm/pull/134
- https://github.com/pulsar-edit/ppm/pull/135

---

As a recap of how to do this:

- Make sure any commits/PRs you want are merged to the main/default branch of `ppm` repo (https://github.com/pulsar-edit/ppm).
- In pulsar core repo's directory, which you should have cloned locally (e.g. by running `git clone --recurse-submodules https://github.com/pulsar-edit/pulsar`)...
   - Run `git submodule update --remote ppm` to fetch and check out the latest commits from the ppm repo on GitHub into your local checkout of the `ppm` submodule in core repo.
   - Run `git add ppm` to commit the changed `ppm` submodule SHA to the core repo's record of the submodule.
   - Run `git commit` to commit the updated submodule in core repo.

In short, that's `git clone --recurse-submodules https://github.com/pulsar-edit/pulsar`, `cd pulsar`, `git submodule update --remote ppm`, `git add ppm`, and `git commit`.

### CODE REVIEW

**Review by confused-Techie — APPROVED** [review]
As each of these changes have been tested thoroughly and tests are passing here, I think we are good to merge, lets do it
